lib/Default/AbstractSmrShip.class.inc

```diff
@@ -983,6 +983,19 @@ abstract class AbstractSmrShip {
 		$thisPlayer->increaseExperience(round($results['TotalDamage']/20)); // 1/20 weapon damage to exp.
 		$thisPlayer->increaseHOF($results['TotalDamage'],array('Combat','Port','Damage Done'), HOF_PUBLIC);
 //		$thisPlayer->increaseHOF(1,array('Combat','Port','Shots')); //in SmrPortt::attackedBy()
+
+		// Change alignment if we reach a damage threshold.
+		// Decrease if player and port races are at war; increase otherwise.
```

Comment:
It should increase if they're at war, not decrease (the code looks to be doing the right thing but the comment should be fixed to avoid confusion)

Reply:
Yeah, that was a silly typo. Fixed. :)